admin.php

```diff
@@ -290,6 +290,9 @@ public function onPluginsInitialized()
             };
         }
 
+        // Load utility class
+        require_once __DIR__ . '/classes/utils.php';
```

Comment:
Use `use Grav\Common\Utils;` at the top, or there's a problem (try installing the Form plugin, for example, and you'll see)